routers/posts.py

```
@@ -1,29 +1,278 @@
+@router.get("/search")
+def search_posts(
+        keyword: str = Query(..., min_length=1),
+):
+    """
+    게시글 검색
+    - 제목 / 내용 / 닉네임 기준 검색
+    - 로그인 필요없음
+    - 검색 결과 전체 반환
+    """
+    posts = load_data("posts")
+    users = load_data("users")
```

Ruff (0.14.13)

Redefinition of unused `users` from line 3: `users` redefined here

(F811)